pkg_defs/ska3-flight-latest/meta.yaml

```diff
@@ -30,17 +23,13 @@ requirements:
     - find_attitude
     - fot-matlab
     - hopper
-    - jobwatch
     - kadi
-    - kalman_watch
     - maude
     - mica
     - parse_cm
-    - perigee_health_plots
     - proseco
     - pyyaks
     - quaternion
-    - ska-sphinx-theme
```

Comment:
This one is the sphinx theme, which is required to build the docs, so I prefer not to remove it. It has not changed in more than three years, I think.

If we remove this, the workflow has to be changed to add it, and users should add it manually before trying to build docs.

Reply:
Sounds like a good reason to just leave it.